### ISSUE TITLE
Does Cypress need an upstream lodash update??

### ISSUE THREAD

**Issue by mkandawire** [cypress-io/cypress#4821]
Installing latest Cypress 3.4.0 and doing `npm audit fix` and `npm audit` gives the following npm audit security report about Cypress

The loadash vulnerability was fixed: https://snyk.io/vuln/npm:lodash

```
$ npm install -D cypress

.
.
.


Cypress 3.4.0 is installed...

npm WARN mobx-react@5.4.4 requires a peer of mobx@^4.0.0 || ^5.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN redoc@2.0.0-rc.8-1 requires a peer of mobx@^4.2.0 || ^5.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN redoc@2.0.0-rc.8-1 requires a peer of styled-components@^4.1.1 but none is installed. You must install peer dependencies yourself.

+ cypress@3.4.0
removed 1 package, updated 3 packages and audited 911190 packages in 13.478s
found 1 high severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details
.
.
.
$ npm audit fix
npm WARN mobx-react@5.4.4 requires a peer of mobx@^4.0.0 || ^5.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN redoc@2.0.0-rc.8-1 requires a peer of mobx@^4.2.0 || ^5.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN redoc@2.0.0-rc.8-1 requires a peer of styled-components@^4.1.1 but none is installed. You must install peer dependencies yourself.

up to date in 6.115s
fixed 0 of 1 vulnerability in 911190 scanned packages
  1 vulnerability required manual review and could not be updated

$ npm audit

                       === npm audit security report ===

┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.12                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ cypress [dev]                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ cypress > lodash                                             │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1065                            │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 1 high severity vulnerability in 911190 scanned packages
  1 vulnerability requires manual review. See the full report for details.
$
```

**Comment by lbphillipvose** [comment]
We've been getting the same issue via GitHubs new security feature, looking at master it has been done but not released?

https://github.com/cypress-io/cypress/pull/4784

**Comment by konekoya** [comment]
Looks like a patch release will be soon released

**Comment by jennifer-shehane** [comment]
Duplicate of https://github.com/cypress-io/cypress/issues/4743

**Comment by jennifer-shehane** [comment]
Released in [`3.4.1`](https://on.cypress.io/changelog#3-4-1).
